frontend/src/app/application-forms/fields/noncommercial-fields.component.spec.ts

```diff
@@ -34,27 +34,33 @@ describe('noncommercial fields', () => {
     })
   );
 
-  it('should be valid if participants is greater than 0', () => {
+  it('should be invalid if paticipants is greater than 0 and spectators is greater than 0 but a sum of less than 75', () => {
     component.noncommercialFields.controls['numberParticipants'].setValue(1);
-    expect(component.noncommercialFields.controls['numberParticipants'].valid).toBeTruthy();
+    component.noncommercialFields.controls['numberSpectators'].setValue(1);
+    component.noncommercialFields.controls['activityDescription'].setValue('test');
+    component.noncommercialFields.controls['locationDescription'].setValue('test');
+    expect(component.noncommercialFields.valid).toBeFalsy();
   });
 
-  it('should be invalid if participants is less than 1', () => {
```

Comment:
we do want to keep this test - we should probably just make it a test of plus for attendees and negative for participants.